src/components/Auth/SignUpForm.js

```diff
@@ -33,7 +33,8 @@ const SignUpForm = ({ toggleActiveForm }) => {
         setErrorMessage('');
       })
       .catch(err => {
-        setErrorMessage(Object.values(err.response.data).join(''));
+        console.log(err);
```

Comment:
Unnecessary console.log

Reply:
I changed this into a `console.error()`; when we implement sentry (https://github.com/codebuddies/frontend/issues/94), I think we can replace this.